mailbox/store/src/test/java/org/apache/james/mailbox/store/probe/SieveProbeTest.java

```diff
@@ -0,0 +1,23 @@
+package org.apache.james.mailbox.store.probe;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SieveProbeTest {
+
+    @Test
+    public void testSanitizeFileNameWithValidInputForSieveProbe() {
+        String validScriptFileNameSiv = "myscript.siv";
+        String validScriptFileNameSieve = "myscript.sieve";
+        assertThat(SieveProbe.sanitizeScriptName(validScriptFileNameSiv)).isEqualTo(validScriptFileNameSiv);
+        assertThat(SieveProbe.sanitizeScriptName(validScriptFileNameSieve)).isEqualTo(validScriptFileNameSieve);
+    }
+
+    @Test
+    public void testSanitizeFileNameWithInvalidInputForSieveProbe() {
+        String validScriptFileName = "myscript.txt";
+        assertThat(SieveProbe.sanitizeScriptName(validScriptFileName)).isEqualTo(validScriptFileName + ".sieve");
+    }
+
```

Comment:
what if there's no extension ? if if finishes by a `.` ? if there's several `.` ?

Reply:
It will just append .sieve in both cases.

Reply:
such questions deserve test case, don't you think ?